e2e/override/proxy-pac-issue-184.yaml

```diff
@@ -0,0 +1,8 @@
+version: "3.8"
+
+services:
+  proxy:
+    environment:
+      FORWARDER_PAC: "/pac.js"
+    volumes:
+      - ./pac/issue-184.js:/pac.js
```

Comment:
CI does not use it. Is is intended?

Reply:
Good catch, I did not add the run script changes.